### PR TITLE
New way to add commandline options

### PR DESCRIPTION
**Pull request by jhaux**
Instead of `--option 'key: value'` you now can do `--key value` or `-key value` to add `value` to the config at `key`.

If possible the `value` strings are converted to python types using `ast.literal_eval`. If not possible the strings are passed as strings.

### CODE REVIEW

**Comment by jhaux** [comment]
Can someone check the pytorch test? It errors in travis, meaning that it does not produce any output in 10 minutes.

**Comment by jhaux** [comment]
I changed the tests to conform with the new way to add options.

File: edflow/edflow [hunk 1]
```diff
@@ -226,20 +228,33 @@ if __name__ == "__main__":
...
+    opt, unknown = parser.parse_known_args()
+    additional_kwargs = parse_unknown_args(unknown)
+
+    if "option" in additional_kwargs or "o" in additional_kwargs:
+        msg = "\n" * 2
+        msg += "=" * 20
+        msg += (
+            "\n\033[93m\033[1m\033[4mThe --option Argument is deprecated!"
+            + "\033[0m\n"
+            + "You kann now pass additional options via --key value "
```
Comment (pesser) [inline edflow/edflow:251]: kann :)

**Review by pesser — APPROVED** [review]
looks good to me but since this changes existing behavior, maybe at least one other contributor can approve before we merge

**Pull request merged**